### DISCUSSION TITLE
Use Portal with Menu

### DISCUSSION THREAD

**Discussion by kimfucious** [mantinedev/mantine#3066] [Q&A] [answered]
Hi there,

I ❤️ Mantine.  Thanks for all the great work.

I'm using the Menu component in a table of records, similar to [this](https://ui.mantine.dev/category/users#users-table).

At times, the menu can have a greater height than the table itself, and thus portions of the menu are clipped.

I tried using the Portal component, but didn't have any luck.

I could add more height to the enclosing element, but that kinda messes up the UI.

Any advise on how to let the menu overflow the containing elements would be most appreciated.

![menu-mantine](https://user-images.githubusercontent.com/26792769/204094899-2bec520a-02c3-4d11-9fdd-f9b5c7a3a3af.jpg)

**Comment by rtivital** [comment, marked as answer]
`<Menu withinPortal />`

**Reply by kimfucious** [reply]
My bad.  

I did a browser search on the Documentation tab and didn't find anything for "portal".

Sorry about that.  I neglected to look under the props tab.
